src/service/mercury/index.test.ts

```diff
@@ -7,8 +7,13 @@ import {
   pubKey,
 } from "../../helper/test-helper";
 import { transformAccountBalances } from "./helpers/transformers";
+import { ERROR_MESSAGES } from ".";
+
+describe.only("Mercury Service", () => {
```

Comment:
i think this `only` was just for testing right?

Reply:
oh thanks, removed in 73d2f5216802f61c2784c80acb665ed24e6f813e